src/main/java/org/mule/extension/http/api/error/HttpError.java

```diff
@@ -155,6 +186,20 @@
     return ofNullable(result);
   }
 
+  /**
+   * Returns whether the error is the parent of a given error represented by a {@code statusCode}. Parent means that the error
+   * belongs to the same hierarchy or kind of error. For example, 4xx are client errors, 5xx are server errors, etc, so a generic
+   * client error should be the parent of a more specific forbidden one (error code 403).
+   * 
+   * @param statusCode
+   * @return
+   *
+   * @since 1.6.0
+   */
+  public boolean isParentOf(int statusCode) {
```

Comment:
maybe `category` is a more appropriate name for the relationship between these status code. This method should be named `isWithinCategory`.

Reply:
You are right, it is a clearer name.